organization_token_integration_test.go

```diff
@@ -38,6 +38,13 @@ func TestOrganizationTokensCreate(t *testing.T) {
 		assert.Nil(t, ot)
 		assert.EqualError(t, err, ErrInvalidOrg.Error())
 	})
+
+	t.Run("without an expiration date", func(t *testing.T) {
```

Comment:
Would recommend also trying to create with an expiry

This will require my above comment (cant link to it for some reason) to be resolved.